py_neuromodulation/nm_database.py

```diff
@@ -0,0 +1,100 @@
+import os
+import sqlite3
+from pathlib import Path
+import numpy as np
+import pandas as pd
+
+class NMDatabase:
+    """
+    Class to create a database and insert data into it.
+    Parameters
+    ----------
+    out_path_root : str
+        The root path to save the database.
+    folder_name : str
+        The folder name to save the database.
+    """
+    def __init__(self, out_path_root, folder_name, csv_path = None):
+        self.out_path_root = out_path_root
+        self.folder_name = folder_name
+        self.db_path = Path(out_path_root, folder_name, "stream.db") 
+        if csv_path is None:
+            self.csv_path = Path(out_path_root, folder_name, f"stream.csv")
+        else:
+            self.csv_path = Path(csv_path)
+
+        if os.path.exists(self.db_path):
```

Comment:
Is this required? So it cannot append data or overwrite?

Reply:
I think appending isn't really an option here, is it? 
By default appending data would mean the db would grow infinitely until deleted. This could lead to errors, especially when working with different settings/features.

Overwriting is basically what we're doing here, as to my understanding when you overwrite you delete a file and create it again. Not deleting the db before will cause errors with column names.

Reply:
And "normal" appending data during the stream (after db init) works by calling the ```insert_data``` method.

Reply:
Yes, exactly. But that's also what it was like with the csv's to my understanding.